hbase-client/src/test/java/org/apache/omid/transaction/TestCellUtils.java

```diff
@@ -137,7 +136,7 @@ public void testCorrectMapingOfCellsToShadowCells() throws IOException {
                    "Should be equal");
 
         // Modify dup shadow cell to have a greater MVCC and check that is replaced
-        HBaseShims.setKeyValueSequenceId((KeyValue) dupCell1WithAnotherValue, 1);
+        ((KeyValue)dupCell1WithAnotherValue).setSequenceId(1);
```

Comment:
Should we be using ExtendedCell rather than KeyValue? (Technically they're both IA.Private, but it at least doesn't couple us to KeyValue's on-heap array-backed implementation)

Reply:
As that would be an effective code change, I think that we should track that in a separate ticket, if you think that we should have that for 1.1. (it's only used in tests)

I originally tried to refactor all the deprecated HBase 1.x APIs as part of this change, but I postponed that to OMID-223 , as it's a lot of work and there are some hairy cases, and I want to get Omid released 1.1 ASAP.